ts/Extensions/Drilldown.ts

```diff
@@ -1062,6 +1064,40 @@ Chart.prototype.drillUp = function (): void {
     fireEvent(chart, 'drillupall');
 };
 
+/**
+ * A function to fade in a group. First, the element is being hidden,
+ * then, using `opactiy`, is faded in. Used for example by `dataLabelsGroup`
+ * where simple SVGElement.fadeIn() is not enough, because of other features
+ * (e.g. InactiveState) using `opacity` to fadeIn/fadeOut.
+ *
+ * @requires module:modules/drilldown
+ *
+ * @param {undefined|SVGElement} [group]
+ * The SVG element to be faded in.
+ */
+function fadeInGroup(
+    this: Highcharts.ChartDrilldownObject,
+    group?: SVGElement
+): void {
+    const animationOptions = animObject(
+        (this.chart.options.drilldown as any).animation
```

Comment:
Do we need `as any` here? Is it possible for `this.chart.options.drilldown` to not exist in this place? If yes then maybe let's create an if statement above?

Reply:
Nice catch! 
It would be another nice-to-improve place. If we could have a Drilldown class, with access to `this.options`, that are always defined in JS (and TS), then we could give up a lot of `as any` castings in the Drilldown module. Right now, this `as any` is one out of seven other cases. 

> Is it possible for this.chart.options.drilldown to not exist in this place?

It's not possible.